Detectors/TRD/base/include/TRDBase/PadParameters.h

```diff
@@ -77,7 +77,8 @@ int PadParameters<T>::getChannel(int col, int row) const
   } else {
     LOG(error) << "Wrong number of channels set: " << mData.size();
   }
-  return -1;
+
+  return 0;
```

Comment:
I would just leave this as is, or why do you prefer a valid return value here?

Reply:
dropped